pkg/commands/container.go

```diff
@@ -57,6 +57,10 @@ func (c *Container) GetDisplayStrings(isFocused bool) []string {
 
 func (c *Container) displayPorts() string {
 	portStrings := lo.Map(c.Container.Ports, func(port types.Port, _ int) string {
+		if port.PublicPort == 0 {
```

Comment:
if port is not exposed on host machine we just want to show the container's port